mimic/canned_responses/flavors.py

```diff
@@ -0,0 +1,25 @@
+"""
+Creates a cannned response for list flavors
+"""
+
+
+def create_flavor(flavor_id):
+    """
+    Populates the flavor template and returns it.
+    """
+    return {"id": "onmetal-" + str(flavor_id),
+            "links": [
+                {"href": "http://this-link-is-not-functional/flavors/" + str(flavor_id),
+                 "rel": "self"},
+                {"href": "http://this-link-is-not-functional/flavors/" + str(flavor_id),
+                 "rel": "bookmark"}],
+            "name": "{0}GB Instance".format(flavor_id)}
```

Comment:
I think there's additional data that's returned by nova's flavor.list API call that's not covered here, like the amount of ram is stored in flavor["ram"]. See https://github.com/rackerlabs/arsenal/blob/master/arsenal/director/onmetal_scout.py#L149